media-tv/plex-media-server/plex-media-server-1.18.3.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+inherit eutils systemd unpacker pax-utils python-single-r1
+
+MINOR_VERSION="2156-349e9837e"
+
+_APPNAME="plexmediaserver"
+_USERNAME="plex"
+_SHORTNAME="${_USERNAME}"
+_FULL_VERSION="${PV}.${MINOR_VERSION}"
+
+URI="https://downloads.plex.tv/plex-media-server-new"
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+SRC_URI="
+	amd64? ( ${URI}/${_FULL_VERSION}/debian/plexmediaserver_${_FULL_VERSION}_amd64.deb )
+	x86? ( ${URI}/${_FULL_VERSION}/debian/plexmediaserver_${_FULL_VERSION}_i386.deb )
+"
+SLOT="0"
+LICENSE="Plex"
+RESTRICT="bindist strip"
+KEYWORDS="-* ~amd64 ~x86"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="dev-python/virtualenv[${PYTHON_USEDEP}]"
+BDEPEND="dev-util/patchelf"
+
+RDEPEND="
+	net-dns/avahi
```

Comment:
You should sort his list, so that acct-* become before net-dns. Then PYTHON_DEPS before or above them.

This looks good, but you said you introduced some USE flag, did it go missing or decided not to add it?